woocommerce/woocommerce
# Remove manually found unicode characters in docs

This PR removes non-regular spaces, quotes, and dashes from docs. This is a requirement to launch the docs site as it doesn't support these unicode characters.

**When reviewing this PR, please verify every single change manually to ensure there are no formatting errors introduced.**

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Removes special characters found within the `/docs` folder. This is currently a draft and will be updated to a full PR once approved and the updated docs manifest is added.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**When reviewing this PR, please verify every single change manually to ensure there are no formatting errors introduced.**
1. Manually review each change to every file modified.
2. Ensure that the content still renders correctly, that it still makes sense, and there are no lingering unsupported special characters. 


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
